assets/dev/js/editor/views/base-container.js

```diff
@@ -155,6 +155,28 @@ module.exports = Marionette.CompositeView.extend( {
 		return widget;
 	},
 
+	onDropEvent( event, options ) {
+		const input = event.originalEvent.dataTransfer.files;
+
+		if ( input.length ) {
+			$e.run( 'editor/browser-import/import', {
+				input,
+				target: this.getContainer(),
+				options: { event, target: { at: options.at } },
+			} );
+
+			return;
+		}
+
+		this.getContainer().view.createElementFromModel(
```

Comment:
Like above, do you need that here?

```suggestion
		this.createElementFromModel(
```

Reply:
Done.